.github/actions/run-nitro-test-node/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Nitro Test Node
+description: 'Checks out the Nitro repository and runs the local test node'
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        repository: OffchainLabs/nitro
+        submodules: 'recursive'
+        path: 'nitro'
+        # TODO: Update with a stable ref once available
```

Comment:
I have a policy of never merging TODOs back to main, and instead always creating a ticket. I find that TODOs in code get forgotten, and also cause clutter because some people use TODOs as part of dev process.

I dont know if you want to continue this policy in the sdk any more though.